doc/plotting.rst

```diff
@@ -0,0 +1,311 @@
+Plotting
+========
+
+Introduction
+------------
+
+The goal of xray's plotting is to make exploratory plotting quick
+and easy by using metadata from :py:class:`xray.DataArray` objects to add
+informative labels.
+
+Xray plotting functionality is a thin wrapper around the popular
+`matplotlib <http://matplotlib.org/>`_ library.
+Matplotlib syntax and function names were copied as much as possible, which
+makes for an easy transition between the two.
+Matplotlib must be installed and working before trying to plot with xray.
+
+For more extensive plotting applications consider the following projects:
+
+- `Seaborn <http://stanford.edu/~mwaskom/software/seaborn/>`_: "provides
+  a high-level interface for drawing attractive statistical graphics."
+  Integrates well with pandas.
+
+- `Holoviews <http://ioam.github.io/holoviews/>`_: "Composable, declarative
+  data structures for building even complex visualizations easily."
+  Works for 2d datasets.
+
+- `Cartopy <http://scitools.org.uk/cartopy/>`_: Provides cartographic
+  tools.
+
+Imports
+~~~~~~~
+
+These imports are necessary for all of the examples.
+
+.. ipython:: python
+
+    import numpy as np
+    import matplotlib.pyplot as plt
+    import xray
+
+One Dimension
+-------------
+
+Simple Example
+~~~~~~~~~~~~~~
+
+Xray uses the coordinate name to label the x axis:
+
+.. ipython:: python
+
+    t = np.linspace(0, np.pi, num=20)
+    sinpts = xray.DataArray(np.sin(t), {'t': t}, name='sin(t)')
+    sinpts
+
+    @savefig plotting_example_sin.png width=4in
+    sinpts.plot()
```

Comment:
do we import seaborn somewhere? the plots here seem to have the seaborn styles....